packages/kamino-sdk/src/constants/pubkeys.ts

```diff
@@ -1,5 +1,6 @@
 import { PublicKey } from '@solana/web3.js';
 
+export const DEFAULT_PUBKEY = new PublicKey(0);
```

Comment:
there is a `Pubkey.default`, you don't need this

Reply:
done